lib/lin.ml

```diff
@@ -113,7 +116,7 @@ struct
           ||
           (* rerun to get seq_sut to same cmd branching point *)
           (let seq_sut' = Spec.init () in
-           let _ = interp_plain seq_sut' (List.rev seq_trace) in
+           let _ = interp_plain_ignore seq_sut' (List.rev seq_trace) in
```

Comment:
Maybe `interp_plain_ignore seq_sut' (List.rev seq_trace);` rather than `let _ = ... in`?

Reply:
Fair point - addressed in 1b50d2f